udaykalra/portfolio/src/main/webapp/style.css

```diff
@@ -67,6 +62,11 @@
   --text-color: #e7e7e7;
 }
 
+#quote-container {
+  margin-top: 20px;
+  color: #fff;
```

Comment:
Missing a few f's? #ffffff

Reply:
GCloud condensed it to 3 f's as they represent the same color.